source/guides/developer.rst

```diff
@@ -2,3 +2,50 @@ Mattermost Developer's Guide
 ============================
 
 Detailed developer documentation is available at `https://developers.mattermost.com <https://developers.mattermost.com>`_.
+||||||| merged common ancestors
```

Comment:
Are these changes unintentional in developer.rst?

Reply:
Whoops, yes!
```suggestion
```